lms/static/js/header/header.js

```diff
@@ -0,0 +1,80 @@
+/*
```

Comment:
Use the standard JSDoc format for comments:

https://edx.readthedocs.io/projects/edx-developer-guide/en/latest/style_guides/javascript-guidelines.html#documentation